spec/bench/create-many-model-class.ts

```diff
@@ -0,0 +1,6 @@
+import { TestClassModel } from "../fixtures/TestClassModel";
```

Comment:
Does this test run? It looks like this is supposed to be a .spec.ts file.

Reply:
No, I don't actually want it to run in CI or as a test. It's a synthetic benchmark that just runs the same code in a loop over and over for profiling / timing. I will move it out of the `spec`  folder to make it clear its not a test.